<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="phoneLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="Newlabel">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nuevo?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewUserLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPasswordLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreatePB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserLE">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>User</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UserLE</tabstop>
  <tabstop>PasswordLE</tabstop>
  <tabstop>LoginPB</tabstop>
  <tabstop>NewUserLE</tabstop>
  <tabstop>phoneLE</tabstop>
  <tabstop>NewPasswordLE</tabstop>
  <tabstop>CreatePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
